_rules/SC1-3-5-autocomplete-valid.md

```diff
@@ -165,76 +190,92 @@ Autocomplete is inappropriate for the type of field.
 <input type="number" autocomplete="email" />
 ```
 
+#### Failed example 6
+
+Autocomplete is not the empty string, but does not have any terms specified.
```

Comment:
Autocomplete is not the empty string > The value of autocomplete is not the empty string